src/main/java/kr/codesqaud/cafe/controller/ReplyController.java

```diff
@@ -0,0 +1,46 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.constant.SessionConst;
+import kr.codesqaud.cafe.dto.Answer;
+import kr.codesqaud.cafe.dto.Result;
+import kr.codesqaud.cafe.dto.SessionDto;
+import kr.codesqaud.cafe.exception.UnauthorizedException;
+import kr.codesqaud.cafe.service.ArticleService;
+import kr.codesqaud.cafe.service.ReplyService;
+import kr.codesqaud.cafe.service.UserService;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class ReplyController {
+
+    private final ReplyService replyService;
+    private final UserService userService;
+    private final ArticleService articleService;
+
+    public ReplyController(ReplyService replyService, UserService userService, ArticleService articleService) {
+        this.replyService = replyService;
+        this.userService = userService;
+        this.articleService = articleService;
+    }
+
+    @PostMapping("articles/{id}/reply")
+    public Answer write(@SessionAttribute(name = SessionConst.LOGIN_MEMBER, required = false)
+                            SessionDto loginUser, @RequestParam String contents, @PathVariable Long id) {
+
+        Long replyId = replyService.write(loginUser.getUserId(), loginUser.getName(), id, contents);
+
+        return new Answer(replyId,loginUser.getUserId(), id, contents);
+
+    }
+
+    @DeleteMapping("articles/{articleId}/reply/{id}")
+    public Result delete(@SessionAttribute(name = SessionConst.LOGIN_MEMBER, required = false)
+                             SessionDto loginUser, @PathVariable Long articleId, @PathVariable Long id) {
+
+        if (replyService.isCreateBy(loginUser.getUserId(), id)) {
+            replyService.delete(id);
+            return new Result("success");
+        }
+        throw new UnauthorizedException();
```

Comment:
일반적으로 UnHappy(예외 처리 등등) 케이스를 if 문 안으로 넣고 early return 하는 구조로 작성하시는게 좀 더 일반적인 방법이여서 고쳐보면 좋을거 같습니다.
```suggestion
        if (!replyService.isCreateBy(loginUser.getUserId(), id)) {
            throw new UnauthorizedException();
        }
        
        replyService.delete(id);
        return new Result("success");
```